examples/notebooks/Getting Started/Tutorial 3 - Basic plotting.ipynb

```diff
@@ -381,19 +381,19 @@
        " 'Positive electrode SEI interfacial current density [A.m-2]',\n",
        " 'X-averaged positive electrode SEI interfacial current density',\n",
        " 'X-averaged positive electrode SEI interfacial current density [A.m-2]',\n",
-       " 'Negative electrode lithium plating concentration',\n",
-       " 'Negative electrode lithium plating concentration [mol.m-3]',\n",
-       " 'X-averaged negative electrode lithium plating concentration',\n",
-       " 'X-averaged negative electrode lithium plating concentration [mol.m-3]',\n",
-       " 'Negative electrode lithium plating thickness',\n",
-       " 'Negative electrode lithium plating thickness [m]',\n",
-       " 'X-averaged negative electrode lithium plating thickness [m]',\n",
-       " 'Loss of lithium to negative electrode lithium plating [mol]',\n",
-       " 'Loss of capacity to negative electrode lithium plating [A.h]',\n",
-       " 'Negative electrode lithium plating interfacial current density',\n",
-       " 'Negative electrode lithium plating interfacial current density [A.m-2]',\n",
-       " 'X-averaged negative electrode lithium plating interfacial current density',\n",
-       " 'X-averaged negative electrode lithium plating interfacial current density [A.m-2]',\n",
+       " 'lithium plating concentration',\n",
```

Comment:
Very minor, but should "lithium" have a capital "L" for consistency with other names?